index.html

```diff
@@ -0,0 +1,170 @@
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Relaycorp - Tackling Neglected Communication Problems</title>
+        <!-- Meta tags for SEO -->
+        <meta name="description" content="Relaycorp is incubating technologies to solve humanity's most pressing and neglected communication challenges, starting with Internet blackouts.">
+        <meta name="keywords" content="Relaycorp, Awala, Letro, VeraId, Despacito, communication, Internet blackouts">
```

Comment:
I think we're already mentioning "Relaycorp" many times here, but missing other keywords:

```suggestion
        <meta name="keywords" content="Awala, Letro, VeraId, Despacito, censorship, Internet blackouts, Internet shutdowns, DDoS attacks">
```